src/data/nat/prime.lean

```diff
@@ -380,6 +380,16 @@ theorem prime.coprime_iff_not_dvd {p n : ℕ} (pp : prime p) : coprime p n ↔ 
 theorem prime.dvd_iff_not_coprime {p n : ℕ} (pp : prime p) : p ∣ n ↔ ¬ coprime p n :=
 iff_not_comm.2 pp.coprime_iff_not_dvd
 
+theorem prime.dvd_of_not_coprime {m n : ℕ} (h : ¬ coprime m n) :
+  ∃p, prime p ∧ p ∣ m ∧ p ∣ n :=
+begin
+  existsi min_fac (gcd m n),
```

Comment:
In mathlib it's generally preferred to use `use` instead of `existsi`

Reply:
Oh! Will work on that.